src/ci/utils.py

```diff
@@ -231,6 +245,88 @@ def cleanLib(moduleName):
     currentDir = currentDirPathBuild(moduleName)
     os.system('make clean -C' + currentDir)
 
+def cleanFolder(folder, additionalFolder):
+    currentDir = currentDirPath(folder)
+    cleanFolderCommand = "rm -rf " + currentDir
+
+    if additionalFolder:
+        cleanFolderCommand += additionalFolder
+
+    out = subprocess.run(cleanFolderCommand, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+    if out.returncode == 0 and not out.stderr:
+        printGreen('[Cleanfolder: PASSED]')
+    else:
+        print(out.stderr)
+        printFail('[Cleanfolder: FAILED]')
+        errorString = 'Error Running Cleanfolder: ' + str(out.returncode)
+        raise ValueError(errorString)
+
+def configureCMake(moduleName, debugMode, withAsan):
+    printHeader("<"+moduleName+">"+headerDic['configurecmake']+"<"+moduleName+">")
+    currentModuleNameDir = currentDirPath(moduleName)
+    currentPathDir = currentDirPathBuild(moduleName)
+
+    if not os.path.exists(currentPathDir):
+        os.mkdir(currentPathDir)
+
+    configureCMakeCommand = "cmake -S" + currentModuleNameDir + " -B" + currentPathDir + " -DCMAKE_BUILD_TYPE=Debug"
```

Comment:
Debug is forced here always, and in the next line if debugMode the same flag is added to the configure command, is that correct?